Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:latest as build-stage
```

Comment:
should we pin to a 1.15 image to match the version in `go.mod`, or do we expect `latest` to support 1.15 forever?

Reply:
I go the other way on this and say that our code will support Golang `latest` forever. But given Dependabot keeping up with that for us, I'll update to 1.15.